docs/03-dev-howtos/17-working-with-amp.md

```diff
@@ -0,0 +1,40 @@
+# Working with Google AMP
+
+**tl;dr - AMP is important, validate your changes (`make validate-amp`)**
+
+## What is AMP?
+AMP (Accelerated Mobile Pages) in Google's own words:
+> AMP is a way to build web pages for static content that render fast. AMP in action consists of three different parts:
+>
+> **AMP HTML** is HTML with some restrictions for reliable performance and some extensions for building rich content beyond basic HTML. The **AMP JS** library ensures the fast rendering of AMP HTML pages. The **Google AMP Cache** can be used to serve cached AMP HTML pages.
+
+## Why do I care?
+At the time of writing, Google is trialling wide roll-outs of AMP blue links in specific regions.  This means that all mobile traffic in these regions coming from Google search results will see AMP versions of our content (if available).  As a result we have to ensure that as much of our content as possible works with AMP.
+
+## Getting started
+* Read the [docs](https://www.ampproject.org/docs/reference/components)
+* Check out the existing AMP work in frontend (in IntelliJ: `shift` `shift` `amp`)
+* Get to know the [tools](#tools)
+
+## <a name="tools"></a>Tools & Hints
+
+### Running AMP
+AMP content can be reached in various ways:
+* **Production** - `amp.theguardian.com/<path>`
+* **Code & local** - append an `amp` parameter to your query string e.g. `localhost:9000/an-article?amp`
+* **Beta** - `beta.amp.theguardian.com/<path>`
```

Comment:
It's how to reach the amp content through fastly beta - so points to PROD, same as `beta.theguardian.com`.